tests/core-manager.js

```diff
@@ -457,6 +458,163 @@ test('unreplicate', async (t) => {
   })
 })
 
+test('deleteData() deletes relevant core storage files', async (t) => {
```

Comment:
apologies for the length of this test. hopefully easy enough to follow but open to recommendations for improving it

Reply:
I'm fine with this, I can read it and it doesn't need to be perfect code. It works and tests what we need it to.